src/index.html

```diff
@@ -11,81 +11,176 @@
+                            <!-- Coloque sua imagem aqui -->
+                            <img src="./assets/images/edu-tech-connect-2.png" alt="Imagem 2">
+                        </div>
+
+                    </div>
+                    <div class="text">
+                        Estamos em busca de voluntários para ensinar habilidades digitais básicas a jovens em
+                        comunidades carentes. Ajude-nos a capacitar essas mentes jovens para um futuro melhor. Sua
+                        experiência é a chave para abrir portas
+                    </div>
+                    <br>
+                    <div class="button-container">
+                        <button class="box-button">
+                            Quero ajudar
+                        </button>
+                    </div>
+                </div>
+                <div class="box">
+                    <div class="img-title">
+                        <div class="title">
+                            Conecte idosos ao mundo digital
+                            <div class="subtitle">SeniorTech</div>
+                        </div>
+                        <div class="image">
+                            <!-- Coloque sua imagem aqui -->
+                            <img src="./assets/images/senior-tech-1.png" alt="Imagem 2">
+                        </div>
+
+                    </div>
+                    <div class="text">
+                        Procuramos voluntários que possam dedicar um pouco do seu tempo para ensinar idosos a usar
+                        smartphones, redes sociais e aplicativos. Sua paciência e conhecimento podem trazer alegria
+                        e inclusão digital às vidas de nossos idosos. Junte-se a nós para fazer a diferença!
+                    </div>
+                    <br>
+                    <div class="button-container">
+                        <button class="box-button">
+                            Quero ajudar
+                        </button>
+                    </div>
+                </div>
             </div>
-            <div class="landing-page-text">
-                <p>Juntos, podemos utilizar a tecnologia para criar um mundo mais solidário!</p>
+            <div class="saiba-voluntariar">
+                <div class="title-voluntariar">
+                    <h3 class="title-saiba-como">Saiba como se voluntariar</h3>
+                    <p class="text-saiba-como">Descubra diversas formas de fazer a diferença em sua comunidade.
+                         Nosso site conecta você a projetos significativos, possibilitando transformar vidas, 
+                         desenvolver habilidades e criar laços duradouros. Junte-se a nós e contribua para um mundo mais unido e solidário.
+                          Inspire-se, atue e transforme.</p>
+                          <div class="button-container">
+                            <button class="box-button">
+                                Quero ajudar
```

Comment:
```suggestion
                                Como começar
```
O texto desse botão é "Como começar"